monkestation/code/datums/traits/negative.dm

```diff
@@ -13,6 +13,7 @@
 
 /datum/quirk/jailbird/proc/apply_arrest(crime_name)
 	var/mob/living/carbon/human/jailbird = quirk_holder
+	jailbird.mind.store_memory("You have the law on your back because of your crime of: [crime_name]!")
```

Comment:
the of feels weird since its going into a : i'd remove  that personally

Reply:
Well, the crimes are written from the perspective of security, so it's hard to really make the tenses line up, as seen in the screenshot.  Without the colon it'd look even more awkward.  I was also trying to imitate the previous text for the trait.